enkit/outputs/commands.go

```diff
@@ -80,6 +81,14 @@ func NewRoot(base *client.BaseFlags) (*Root, error) {
 	return rc, nil
 }
 
+func (rc *Root) validate(cmd *cobra.Command, args []string) error {
+	_, _, err := rc.BaseFlags.IdentityCookie()
```

Comment:
I'm not sold on this approach because:

* I don't think this code handles the case when the user has stale credentials (only the case when the user has no credentials)
* This means a not-logged-in user can't use this command, even if they wanted to manually fill out the required flags (though in this specific case that is moot, because probably the command will require login for a tunnel)

Either we do this approach but replace the IdentityCookie check with a stronger indicator of whether the creds actually work (is there an endpoint we can hit?) and/or we ensure that config loading throws errors/warnings appropriately as we fail to fetch includes that require auth.